src/main/java/com/oltpbenchmark/benchmarks/templated/util/TemplatedValue.java

```diff
@@ -0,0 +1,112 @@
+package com.oltpbenchmark.benchmarks.templated.util;
+
+import java.util.Random;
+
+import com.oltpbenchmark.distributions.ScrambledZipfianGenerator;
+import com.oltpbenchmark.distributions.ZipfianGenerator;
+
+/**
+ * This class is used to store information about the values
+ * used in templated benchmarks. It can hold static values but also
+ * generators for value distributions
+ */
+public class TemplatedValue {
+    ValueGenerator distribution;
+    Long min;
+    Long max;
+    Long seed;
+    String value;
+    String minS;
+    String maxS;
+
+    public final Object generatorObject;
+
+    /**
+     * 
+     * @param dist  The desired value distribution
+     * @param min   Minimum value. Default is 0
+     * @param max   Maximum value. Default is 1
+     * @param seed  The seed for the random generator. Default is 0
+     * @param value Value that is used if no distribution is given
+     */
+    public TemplatedValue(String dist, String min, String max, String seed, String value) {
+        this.min = min == null ? 0 : Long.parseLong(min);
+        this.max = max == null ? 1 : Long.parseLong(max);
+        this.seed = seed == null ? 0 : Long.parseLong(seed);
+        this.value = value;
```

Comment:
Is `value` valid when `distribution` is also provided?

Reply:
General point is you probably need some validation of the values being passed in (e.g., all expected ones are present or not, correct relative ranges, etc.)